### PR TITLE
Update Quality Declaration to match Level 2

### PR DESCRIPTION
**Pull request by Blast545**
As the title says. Also added the Quality Claim in the README.md file.

Signed-off-by: Jorge Perez <jjperez@ekumenlabs.com>

### CODE REVIEW

File: QUALITY_DECLARATION.md [hunk 1]
```diff
@@ -46,7 +46,13 @@ All pull requests will be peer-reviewed, check [ROS 2 Developer Guide](https://i
 
 ### Continuous Integration [2.iv]
 
-This package is not being tested under ROS2 CI.
+All pull request must pass CI on all [tier 1 platforms](https://www.ros.org/reps/rep-2000.html#support-tiers)
```
Comment (hidmic) [inline QUALITY_DECLARATION.md]: @Blast545 
```suggestion
All pull requests must pass CI on all [tier 1 platforms](https://www.ros.org/reps/rep-2000.html#support-tiers)
```

File: QUALITY_DECLARATION.md [hunk 2]
```diff
@@ -46,7 +46,13 @@ All pull requests will be peer-reviewed, check [ROS 2 Developer Guide](https://i
...
+
+Currently nightly results can be seen here:
```
Comment (hidmic) [inline QUALITY_DECLARATION.md]: @Blast545 
```suggestion
Current nightly results can be seen here:
```

**Review by hidmic — APPROVED** [review]
Minor comments here and there.

File: QUALITY_DECLARATION.md [hunk 3]
```diff
@@ -46,7 +46,13 @@ All pull requests will be peer-reviewed, check [ROS 2 Developer Guide](https://i
 
 ### Continuous Integration [2.iv]
 
-This package is not being tested under ROS2 CI.
+All pull requests must pass CI on all [tier 1 platforms](https://www.ros.org/reps/rep-2000.html#support-tiers)
```
Comment (brawner) [inline QUALITY_DECLARATION.md]:
```suggestion
All pull requests must pass CI on all [tier 1 platforms](https://www.ros.org/reps/rep-2000.html#support-tiers).
```

**Review by brawner — APPROVED** [review]

**Review by chapulina — APPROVED** [review]

**Pull request merged**